### ISSUE TITLE
CANT MAKE IT WORK ON UBUNTU

### ISSUE THREAD

**Issue by LostName1** [mangunowsky/MFBotDocker#11]
I am trying to set up the bot on my ubuntu server but there is no way for it. Can I get any help? Thanks :D

**Comment by mangunowsky** [comment]
Hello! 

Please describe what the issue is with as much details as you can.

**Comment by LostName1** [comment]
Sure,

I followed the guide step by step.

1st Cloned the repo
2nd Pasted on the folder the Acc.ini created with the desktop version
3rd Built docker image
4th Run docker container

Until this point everything works as expected

5th Try entering the web interface

The problem comes here, there is no way to make the web interface work, I tried 8050:8050 tried 8050:80 80:8050 and many more ports but I could not make it work.

I can see the next error on the logs: pycurl.error: (7, 'Failed to connect to 127.0.0.1 port 6969: Connection refused')

**Comment by mangunowsky** [comment]
Make sure you have correctly set up remote access. Here's how it looks for me:

![Screenshot from 2024-04-18 12-29-46](https://github.com/mangunowsky/MFBotDocker/assets/74178234/8968c9c9-d1b3-43d0-8482-6a3a935258e0)

Update `Acc.ini` file, rebuild the container and try again. If you followed instruction, you should be able to access web interface under this address `http://localhost:8050/` from your browser.

**Comment by LostName1** [comment]
> Make sure you have correctly set up remote access. Here's how it looks for me:
> 
> ![Screenshot from 2024-04-18 12-29-46](https://private-user-images.githubusercontent.com/74178234/323566848-8968c9c9-d1b3-43d0-8482-6a3a935258e0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM0MzczNDQsIm5iZiI6MTcxMzQzNzA0NCwicGF0aCI6Ii83NDE3ODIzNC8zMjM1NjY4NDgtODk2OGM5YzktZDFiMy00M2QwLTg0ODItNmEzYTkzNTI1OGUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE4VDEwNDQwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU0NmUxZjdlODViZTQxM2M0ODA0ODYwYjBjYzVlOTc0NjJiZmQ1MjNkN2U2M2I3MDE0ZTY4Y2VmNGYwOWY3N2MmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uUknNw0_7UjnRLjuFaTjltgs7O8FmONkSIZCoELxx24)
> 
> Update `Acc.ini` file, rebuild the container and try again. If you followed instruction, you should be able to access web interface under this address `http://localhost:8050/` from your browser.

is it possible to enable this via command? It is ubuntu with user interface, only console

**Comment by mangunowsky** [comment]
> is it possible to enable this via command? It is ubuntu with user interface, only console

Yes, there are couple of lines you can edit in `Acc.ini` with text editor.

For example:
```
REMOTE=1
REMOTE.DEBUG=0
REMOTE.IP=127.0.0.1
REMOTE.MASTER=admin
REMOTE.MASTERPASSWORD=admin
REMOTE.PORT=6969
REMOTE.USER
```

**Comment by LostName1** [comment]
> > is it possible to enable this via command? It is ubuntu with user interface, only console
> 
> Yes, there are couple of lines you can edit in `Acc.ini` with text editor.
> 
> For example:
> 
> ```
> REMOTE=1
> REMOTE.DEBUG=0
> REMOTE.IP=127.0.0.1
> REMOTE.MASTER=admin
> REMOTE.MASTERPASSWORD=admin
> REMOTE.PORT=6969
> REMOTE.USER
> ```

But how can I enable remote access vía console?

**Comment by mangunowsky** [comment]
I'm not sure if you can do that via console. In Acc.ini `REMOTE=1` is what you are looking for. Edit the file and rebuild the container.

**Comment by LostName1** [comment]
I have changed remote=1 in Acc.ini file and I keep getting this error


pycurl.error: (7, 'Failed to connect to 127.0.0.1 port 6969: Connection refused')

**Comment by mangunowsky** [comment]
Sorry it took so long, I needed to fix some things to be able to run Windows VM. 

The error you mentioned occurs typically when you failed to configure remote access properly. Please make sure remote access is enabled, you set admin login and password and you are trying to connect over correct port.

Also please include more details about your setup like bot version used to create Acc.ini, system version, etc.
